module-1/lab-api-scavenger-game/your-code/API_Challenge_1.ipynb

```diff
@@ -0,0 +1,1479 @@
+{
```

Comment:
Good Job!!
Remember to normalize your JSON always because in a get response text you could have nested JSONs.